src/Raven.Studio/typescript/commands/database/documents/getDocumentsPreviewCommand.ts

```diff
@@ -62,8 +44,10 @@ class getDocumentsPreviewCommand extends commandBase {
                 continuationToken: dto.ContinuationToken
             };
         };
-        
+
+        const args = this.getArgsToUse(this.continuationToken);
```

Comment:
do we need to pass token to this function?

Reply:
removed